src/mui/list/InfiniteList.js

```diff
@@ -108,9 +108,25 @@ export class InfiniteList extends List {
         );
         return (
             <Paper style={{ background: 'transparent' }} zDepth={0}>
-                <div style={styles.header}>
-                    <ViewTitle title={titleElement} style={styles.title} />
+                <div>
```

Comment:
That div contains all the header. Should she take the prop style.header ?

Reply:
No, or maybe I should rename the style

Reply:
I guess son it's a little bit confusing